swift/swift.py

```diff
@@ -41,7 +41,7 @@ class AppInfo:
         pos: Position on the main window; refer to Qt.DockWidgetArea enum.
           Should be one of "left", "right", "top", or "bottom", case-sensitive.
           Otherwise, defaults to Qt.LeftDockWidgetArea.
-        bus: Buses which the app subscribes to.
+        channel: Channeles which the app subscribes to.

```

Comment:
May be you have simply replaced the words 😄
Please change it to "Channels".

Reply:
Ah.. I checked the other files, but I don't think I can do it here..😂

Reply:
I think the other "channels" are fine!
Channeles -> Channels